src/main/java/com/minecolonies/coremod/entity/ai/citizen/composter/EntityAIWorkComposter.java

```diff
@@ -81,6 +82,11 @@
     private final static VisibleCitizenStatus COMPOST =
       new VisibleCitizenStatus(new ResourceLocation(Constants.MOD_ID, "textures/icons/work/composter.png"), "com.minecolonies.gui.visiblestatus.composter");
 
+    /**
+     * it makes it work. go away
+     */
+    private final Random random = new Random();
```

Comment:
Why not use the one built into the citizen's entity? worker.getRandom() should work where you use this.

Reply:
this

Reply:
Change it to worker.getRandom() here? or at the actual podzol chance calc?

Reply:
You could remove the declaration (`private final Random random = new Random();`) and related javadoc, and in the podzol chance calculation, use `if (((worker.getRandom().nextInt(100)) + 1) <= (5 * (1` + `if (((random.nextInt(100)) + 1) <= (5 * (1 +`.

Do watch out that it won't clean up the import statements automatically.